android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/identitycredential/IdentityCredentialItem.kt

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.prompts.identitycredential
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Text
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import mozilla.components.ui.colors.PhotonColors
+
+/**
+ * List item used to display an IdentityCredentials item that supports clicks
+ *
+ * @param title the Title of the item
+ * @param description the Description of the item
+ * @param modifier The modifier to apply to this layout.
+ * @param onClick Invoked when the item is clicked.
+ * @param beforeItemContent An optional layout to display before the item
+ *
+ */
+@Composable
+internal fun IdentityCredentialItem(
+    title: String,
+    description: String,
+    modifier: Modifier = Modifier,
+    onClick: () -> Unit,
+    beforeItemContent: (@Composable () -> Unit)? = null,
+) {
+    // Used to propagate the ripple effect to the whole row
+    val interactionSource = remember { MutableInteractionSource() }
+
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .clickable(
+                onClick = onClick,
+                interactionSource = interactionSource,
+                indication = rememberRipple(color = PhotonColors.Black),
```

Comment:
you might need an `if` here to have colors for light and dark mode

Reply:
Maybe we could just receive the colour as parameter and the consumer of the component can decide when to pass the dark or light version ?